### DISCUSSION TITLE
Can I resize Image and remove the container around the text ?

### DISCUSSION THREAD

**Discussion by fdclouchin** [burhanrashid52/PhotoEditor#536] [Q&A]
I would like to ask if its possible to resize the image and remove the container of the text after adding? Can I also add background for the text?

**Comment by burhanrashid52** [comment]
For text styling, you can check this [TextStyleBuilder](https://github.com/burhanrashid52/PhotoEditor/blob/master/photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextStyleBuilder.kt). If this does not have the property you looking for then you can raise a PR adding those parameters in the builder.
For images, we don't have many options. Maybe having a similar `ImageStyleBuilder` might help.

**Reply by fdclouchin** [reply]
Thanks.

**Comment by davidfrancis597** [comment]
<html>
<body>

Yes, you can resize an image and remove the container around the text using photo editing software like Photoshop or online tools like Canva.<a href="https://photoeditingindia.com/sky-change-services/">outsource real estate sky change services</a> Resize the image by adjusting its dimensions or crop it to fit your needs. To remove the text container, simply use the selection tool to delete or hide it, ensuring a clean and seamless design.
--


<br class="Apple-interchange-newline">
</body>
</html>
